smartcontractkit/external-adapters-js
# Feat/DF-19863 composite PoR using coinbase-prime wallets endpoint

## Closes #[DF-19863](https://smartcontract-it.atlassian.net/browse/DF-19863)

## Description
Added `wallets` endpoint to coinbase-prime adapter
Added this new coinbase-prime `wallets` endpoint as an option in the proof-of-reserves composite stack as address list fetcher

## Changes
- added `wallets` endpoint to coinbase-prime adapter
- added necessary plumbing to set it up in the proof-of-reserves composite EA



## Steps to Test

1. (from root) `yarn test coinbase-prime`
2. using proof-of-reserves setup with coinbase-prime and por-indexer, ensure the coinbase-prime `balance` endpoint returns data that is roughly in line with the final composite stack result

## Quality Assurance



[DF-19863]: https://smartcontract-it.atlassian.net/browse/DF-19863?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
